ygot/render.go

```diff
@@ -1147,6 +1150,17 @@ func constructJSONValue(field reflect.Value, parentMod string, args jsonOutputCo
 		if args.jType == RFC7951 {
 			value = writeIETFScalarJSON(value)
 		}
+	case reflect.Bool:
+		// A non-pointer field of type boolean is an empty leaf within the YANG schema.
+		// For RFC7951 this is represented as a null JSON array (i.e., [null]), if the
+		// case that this leaf is present in internal JSON, then it is rendered as 'true'
+		// only if the leaf is set, otherwise it is nil.
```

Comment:
For RFC7951 this is represented as a null JSON array (i.e., [null]). 
For internal JSON if the leaf is present and set it is rendered as 'true', or as nil otherwise.
(Not sure if that's the correct description, but something like that...)

Reply:
Done.